### DISCUSSION TITLE
Panopticon (WIP)

### DISCUSSION THREAD

**Discussion by SniperGoth** [redeclipse/base#1369] [Maps and Content Showcase]
A simple, circular layout map set inside a abandoned panopticon facility, with various levels for playing, and a tunnel below.
Overview:
![20221001162317](https://user-images.githubusercontent.com/24359960/193425376-1241de59-e79c-42ea-8d02-22c34c942701.png)

Tunnel entrance:
![20221001162454](https://user-images.githubusercontent.com/24359960/193425407-1e6494a7-547d-4bc8-85d5-fde4b832b8ad.png)

Tunnel inside (leads to the inside of the central tower, has two mines):
![20221001162145](https://user-images.githubusercontent.com/24359960/193425426-7923f842-f6f9-421d-9705-aa9c32ec59b9.png)

Ground (barely if any cover at all, no man's land):
![20221001162934](https://user-images.githubusercontent.com/24359960/193425473-58fc2145-54fb-49c1-9012-034d2a2db049.png)

Short wall overview, on the second floor:
![20221001162958](https://user-images.githubusercontent.com/24359960/193425485-5b8e70ce-3601-4d17-a4c3-4f0b69bdfd6f.png)

I started today the creation of this map, so it's still pretty much a WIP, with very basic lighting and even more basic texturing, but i wanted to get this out as soon as possible for people to try it out.
4 grenades, 2 mines, 1 rocket, but no weapon spawns (yet), waypoints are all setup for bots.
Works on deathmatch, defend and control (the control point is inside the main structure, you need to take the tunnels to reach it, or a precise slide jump).
Any feedback is appreciated.
Works on 2.0, and git version.
Small hotfix: forgot to add ladder textures for better visualization.
![20221001165321](https://user-images.githubusercontent.com/24359960/193426100-58ddd638-6bee-4c0c-913d-4f8fe01de8bd.png)
Hotfix 2: forgot to increase the minimum number of bots from 4 to 8.
[panopticon-prototype-map.zip](https://github.com/redeclipse/base/files/9691120/panopticon-prototype-map.zip)

**Comment by MirceaKitsune** [comment]
Looked at it briefly. A bit hard to say at this stage, given it's still rather empty so it feels very large and open. I found the tunnels are easy to get lost in as it stands, could use more references and uniqueness perhaps. I can probably get a better feel once there's more complexity but overall it's not a bad layout!
